com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -490,7 +490,7 @@ private void ProcessNetworkBehaviour(TypeDefinition typeDefinition, string[] ass
                         m_MainModule.AssemblyReferences.Remove(reference);
                         break;
                     }
-                    catch (Exception e)
+                    catch (Exception)
```

Comment:
Can we catch just the exceptions we  expect to throw in `AssemblyReferences.Remove`?  Else it makes debugging really tricky as it catches exceptions in the handler

Reply:
This was merged yesterday and it doesn't change any functionality. I'm all for now specific exception handling but it needs to be another work item